packages/block-editor/src/components/block-canvas/style.scss

```diff
@@ -6,4 +6,5 @@ iframe[name="editor-canvas"] {
 	background-color: transparent;
 	// Handles transitions between device previews
 	@include editor-canvas-resize-animation;
+	background-color: $gray-300;
```

Comment:
I don't remember exactly, but I think we wanted to have `transparent` background (now you have two `background-color` rules). Maybe something with the background in theme.json if defined? 🤔

Reply:
Actually, the issue seems in trunk too, so there must be something else going on. The regression is not caused by this PR. cc @jasmussen